src/_includes/partials/helpus.njk

```diff
@@ -1,33 +1,35 @@
-<hr />
+<section class="py-400 mt-500 bt-sm text-secondary">
+	
+	<h2 class="text-primary">
+	{{ helpus[locale].heading }}
+	</h2>
 
-<h2 class="mt-500 mb-400">
-  {{ helpus[locale].heading }}
-</h2>
+	<p class="mt-400 text-primary">
```

Comment:
You can remove `text-primary` class here after removing the `text-secondary` from the section as it won't be needed.

Reply:
For the border too.

Reply:
Removed my changes